src/server/NetworkingServer.cpp

```diff
@@ -1,71 +1,107 @@
 #include "NetworkingServer.hpp"
 
-NetworkingServer::NetworkingServer() {
-    mClientIdCounter = 0;
-    mCurrTick = 0;
+NetworkingServer::NetworkingServer():mCurrTick(0) {
```

Comment:
I don't know why mCurrTick was moved here, this is a lot harder to read

Reply:
Codeclimate (cppcheck) complained about this being constructed in the NetworkingServer constructor rather than before. We can't initialize it in the header file AFAICT because "You cannot use copy-initialization, because std::atomic_bool is not copy-constructible" so I have to do an inline list _direct_ initialization.

See https://stackoverflow.com/a/15751048

Read this: https://stackoverflow.com/a/15751048